resources/settings.xml

```diff
@@ -33,6 +33,13 @@
         <setting id="hide_types" type="enum" label="30082" lvalues="30083|30084" default="1" />
         <setting id="geo_region" type="select" label="30085" lvalues="30074|30306|30309|30308|30307|30303|30304|30301|30305|30302" />
         <setting id="geo_label" type="text" label="30073" enable="false" />
+
```

Comment:
This part also needs to be in the settings template: https://github.com/retrospect-addon/plugin.video.retrospect/blob/master/resources/data/settings_template.xml otherwise it will be overwritten during build.